dask_pytorch/dispatch.py

```diff
@@ -31,13 +31,13 @@ def run(client: Client, pytorch_function: Callable, *args, **kwargs):
     futures = [
         client.submit(
             dispatch_with_ddp,
-            pytorch_function = pytorch_function,
-            master_addr = host,
-            master_port = port,
-            rank = idx,
-            world_size = world_size,
-            *args,
-#             workers=[w],
+            pytorch_function,
+            host,
+            port,
+            idx,
+            world_size,
+            "nccl",
+            *args, 
```

Comment:
```suggestion
            pytorch_function=pytorch_function,
            master_addr=host,
            master_port=port,
            rank=idx,
            world_size=world_size,
            backend="nccl",
            *args, 
```

I think this type of issue could be avoided in the future by using keyword arguments wherever possible

Reply:
I had the names listed before, and then strangely it broke the code- I'll try again though, I was wondering if the naming was differently handled because this is using `client.submit` instead of running the function directly.

Reply:
To be more exact, backend has a default value provided, so I had named everything and then just left that one off- but it broke. I guess the default value needs to be explicitly stated?

Reply:
Ok, checked- looks like it worked! I made the change.